apps/web/components/blog/mdx-renderer.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import { createMDXComponents } from "@/lib/blog/mdx/components/mdx-components";
+import { cn } from "@/lib/utils";
+import { MDXRemote, type MDXRemoteSerializeResult } from "next-mdx-remote";
```

Comment:
next-mdx-remote hasn't been updated in 5+ years and also has some known security vulnerabilities. Why not `@next/mdx` ?